packages/manager/modules/pci/src/components/project/images-list/images-list.service.js

```diff
@@ -26,6 +26,14 @@ export default class ImagesList {
     this.$http = $http;
     this.OvhApiCloudProjectImage = OvhApiCloudProjectImage;
     this.OvhApiCloudProjectSnapshot = OvhApiCloudProjectSnapshot;
+    this.REGEX_PREFIX_BAREMETAL = REGEX_PREFIX_BAREMETAL;
+  }
+
+  removeBaremetalPrefix(imageType) {
+    if (imageType.match(this.REGEX_PREFIX_BAREMETAL)) {
```

Comment:
directly use REGEX_PREFIX_BAREMETAL in match function.

Reply:
As seen together, a function in a service cannot not have "this. (except for static functions). Without it, we have a lint message.